File: SPACE TRIANGLE PATTERN/code8.c
#include <stdio.h>

void main (){

	int row ;
	printf("Enter no. of rows : ");
	scanf("%d",&row);


	int temp = row * row;
	for(int i = 1 ; i <= row ; i++){

		for(int a = 0 ; a < row - i ; a++){

			printf(" \t");
		}

		for(int j = 1; j <= i ; j++){

			printf("%d\t",j*i);

		}

		printf("\n\n");

	}


}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/9/code8.java

import java.util.Scanner;

class NestedDemo{

	public static void main(String[] args){

		Scanner s= new Scanner(System.in);
		System.out.print("Enter no of rows : ");
		int row = s.nextInt();

		int temp = row*row;
		for(int i = 0; i < row ; i++){
			
			for(int j = 0; j < row ; j++){
					
				if(i % 2==0){
					System.out.print((char)(row - j +64)+"		");

				}else{
					System.out.print((j+1)+"		");

				}
			}

				System.out.print("\n\n");

		}

	}

}
